### DISCUSSION TITLE
Termo: fisgar as iscas (phishing)

### DISCUSSION THREAD

**Discussion by celsobessa** [infuse-l10n-pt-br/Infuse#19] [Glossário]
Baseado em conversas, casos anedóticos de uso com profissionais e estudantes, proponho usar _fisgar a isca_ como tradução de _phising_.

Por exemplo: "por que [macros] são tão populares em ataques de _fisgar a isca_ (_phishing_, em inglês) para infectar computadores em comparação com outras técnicas."

**Comment by contraexemplo** [comment]
Acho que não é uma boa tradução de phishing em si, mas é uma boa maneira de comunicar o efeito desejado do phishing. Eu diria que "pescar o alvo", ou "pescar vítimas" seriam traduções mais diretas do que a palavra _phishing_ quer transmitir.

**Reply by contraexemplo** [reply]
Pensando mais um pouco enquanto escrevo o glossário: pescar a atenção de alguém? Eu não gosto muito de ficar usando "vítima", e a moeda aqui que um atacante procura é a atenção das pessoas.
